app/components/Transaction/index.jsx

```diff
@@ -0,0 +1,92 @@
+/**
+ *
+ * Transaction
+ *
+ */
+
+import React from 'react';
+import { Col, Row } from 'reactstrap';
+import styled from 'styled-components';
+
+import ArrowIcon from 'react-icons/lib/io/arrow-right-c';
+import tokenLogo from 'images/token31.png';
+import './transaction.scss';
+
+
+const IMG = styled.img`
+  width: 48px;
+  height: 48px;
+  margin-right: 7px;
+`;
+
+class Transaction extends React.PureComponent { // eslint-disable-line react/prefer-stateless-function
+  render() {
+    return (
```

Comment:
note: these will need to come from api calls as well

Reply:
Yes, that data is mocked until to consume the API.